src/components/ItaliaTheme/View/Commons/PageHeader/PageHeaderEventDates.jsx

```diff
@@ -25,11 +25,31 @@ const PageHeaderEventDates = ({ content, moment, rrule }) => {
   Moment.locale(intl.locale);
 
   const rrulestr = rrule.rrulestr;
+
+  const rruleSet = content.recurrence
+    ? rrulestr(content?.recurrence, {
+        compatible: true, //If set to True, the parser will operate in RFC-compatible mode. Right now it means that unfold will be turned on, and if a DTSTART is found, it will be considered the first recurrence instance, as documented in the RFC.
+        forceset: true,
+      })
+    : null;
+
+  const getRealEventEnd = (content) => {
+    let actualEndDate = content.end;
+    if (content.recurrence) {
+      actualEndDate = rruleSet.rrules()[0].options.until;
+    }
+    return actualEndDate;
+  };
```

Comment:
questa sparirà perchè lo sposterai in un file che sta sotto src/helpers e lo importerai